extensions/amp-sidebar/1.0/test/test-component.js

```diff
@@ -681,7 +681,7 @@ describes.sandboxed('Sidebar preact component', {}, (env) => {
       document.body.removeChild(target);
     });
 
-    it('toolbar target receives content when media query is true', () => {
+    it('toolbar target receives expected content from toolbar', () => {
```

Comment:
Let's add a test for media query and css ident sanitization.

Reply:
Yep, definitely, will do!